packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextView.java

```diff
@@ -680,16 +666,37 @@ protected boolean dispatchHoverEvent(MotionEvent event) {
     return super.dispatchHoverEvent(event);
   }
 
-  private void applyTextAttributes() {
-    // Workaround for an issue where text can be cut off with an ellipsis when
-    // using certain font sizes and padding. Sets the provided text size and
-    // letter spacing to ensure consistent rendering and prevent cut-off.
-    if (!Float.isNaN(mFontSize)) {
-      setTextSize(TypedValue.COMPLEX_UNIT_PX, mFontSize);
+  public void setLetterSpacingPt(float letterSpacingPt) {
```

Comment:
Why was this changed to `pt`? Input should be SP/dips.

Reply:
> Why was this changed to `pt`? Input should be SP/dips.

It's just a bad name, I'll change it.